nengo_gui/components/bg_plot.py

```diff
@@ -0,0 +1,68 @@
+import nengo
+
+from nengo_gui.components.value import Value
+
+
+class BGPlot(Value):
+    """The server-side system for the SPA Basal Ganglia plot."""
+
+    # the parameters to be stored in the .cfg file
+    config_defaults = Value.config_defaults.copy()
+    config_defaults["show_legend"] = True
+
+    def __init__(self, obj, **kwargs):
+        super(BGPlot, self).__init__(obj)
+
+        args = kwargs["args"]
+
+        # default legends to show
+        self.def_legend_labels = args["legend_labels"]
+
+        # the item to connect to
+        self.probe_target = args["probe_target"]
+
+        self.label = "bg " + self.probe_target
+
+    def attach(self, page, config, uid):
+        super(Value, self).attach(page, config, uid)
```

Comment:
The difference between this calling of the constructor and not defining it at all, is that in this call, I don't assign a label. If this were not defined, a label, which I do not want, would be assigned.